tools/policy-gen/bootstrap/root.go

```diff
@@ -172,15 +172,7 @@ message {{ .name }} {
     skip_registration : false,
   };
 
-  message Spec {
-    {{- if not .useCustomMatching }}
-    // TODO @lobkovilya add protobuf for standard matchers
-    {{- end }}
```

Comment:
To me with customMatching you can straight up add:

```
  // TargetRef is a reference to the resource the policy takes an effect on.
  // The resource could be either a real store object or virtual resource
  // defined inplace.
  kuma.common.v1alpha1.TargetRef targetRef = 1;

  message From {
    // TargetRef is a reference to the resource that represents a group of
    // clients.
    kuma.common.v1alpha1.TargetRef targetRef = 1;

    // Default is a configuration specific to the group of clients referenced in
    // 'targetRef'
    Conf default = 2;
  }


  message To {
    // TargetRef is a reference to the resource that represents a group of
    // clients.
    kuma.common.v1alpha1.TargetRef targetRef = 1;

    // Default is a configuration specific to the group of clients referenced in
    // 'targetRef'
    Conf default = 2;
  }

  // From is a list of pairs – a group of clients and action applied for it
  repeated From from = 2;
  // To is a list of pairs – a group of clients and action applied for it
  repeated To to = 2;
  
  message Conf {
    // TODO add match specific fields
  }
```

(And the import for targetRef).

If you don't put this in bootstrap these are things that are going to be copy/pasted by everyone creating a policy.

Reply:
Agree, I'll do that in the follow-up PR